app/components/user-search.cjsx

```diff
@@ -12,7 +12,7 @@ module.exports = React.createClass
       apiClient.type('users').get display_name: value, page_size: 10
         .then (users) =>
           opts = for user in users
-            { value: user.id, label: user.display_name }
+            { value: user.id, label: "@#{ user.login }: #{ user.display_name }" }
```

Comment:
I was basically just mimicking most @-mention systems (e.g. github) where they show something like

*@user1* _John Doe_
*@user2* _Jane Doe_

But I guess since display names are unique, it doesn't matter that much.